docs/source/tutorial.rst

```diff
@@ -117,39 +123,73 @@ build, run the following inside vagrant:
 
   $ /project/sawtooth-core/bin/build_all
 
-Running validator
-=================
+Validator Start-up Process
+==========================
+
+Create Genesis Block
+--------------------
+
+The first time that a validator is run, the validator needs to ingest a batch
+of transactions that set up some initial values for use by the first block in
+the block-chain, called the "genesis block."
+
+To create the genesis block, log in to the development environment with the
+command `vagrant ssh` and run the following command:
+
+.. code-block:: console
+
+  $ sawtooth admin genesis
```

Comment:
Need to add the flow for creating the initial transactions like config. Something along these lines: 1. create config transactions 2. <create other initializing transactions e.g. poet, mktplace,...>. 3. create the genesis batch consuming transactions from 1 and 2.